api/src/main/java/org/jmisb/api/klv/st0601/UasEnumeration.java

```diff
@@ -0,0 +1,52 @@
+package org.jmisb.api.klv.st0601;
+
+import java.util.Collections;
+import java.util.Map;
+import static org.jmisb.api.klv.st0601.IcingDetected.DISPLAY_VALUES;
+
+public abstract class UasEnumeration implements IUasDatalinkValue {
+
+    protected byte value;
+
+    abstract Map<Integer, String> getDisplayValues();
+
+    abstract String getDisplayName();
```

Comment:
I am wondering if we should push this up to IUasDatalinkValue, what are your thoughts? If we think it makes sense let's create a new issue for that.

Reply:
I added it here because it helped to make the error messages better. However I think it could help for re-usable code. I'll add an issue to at least evaluate it further: #42